pyresample/resampler.py

```diff
@@ -132,3 +148,166 @@ def _create_cache_filename(self, cache_dir=None, prefix='',
         hash_str = self.get_hash(**kwargs)
 
         return os.path.join(cache_dir, prefix + hash_str + fmt)
+
+
+def resample_blocks(funk, src_area, src_arrays, dst_area,
```

Comment:
This function is kind of big. Can it be broken up?

Reply:
Done.

Reply:
`func` versus `funk`?

Reply:
This is a lot of code for `resample_blocks`. In a future PR or maybe my pyresample 2.0 work we should/could maybe move this to a separate module. Thoughts?